docs/app-integration-development/20-Act-as-App-Token.md

```diff
@@ -2,7 +2,7 @@
 tags: [app-integration-development]
 ---
 
-# Scoped OAuth
+# Act as App Tokens
```

Comment:
"Act as App" vs "Act as User" tokens is a little awkward, maybe App OAuth tokens vs User OAuth tokens?

Reply:
Made a pass at this. See what you think.